model-optimizer/extensions/ops/elementwise_test.py

```diff
@@ -0,0 +1,104 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import unittest
+
+import numpy as np
+
+from extensions.ops.elementwise import Round
+from mo.graph.graph import Node
+from mo.utils.unittest.graph import build_graph
+
+
+class TestElementwiseOp(unittest.TestCase):
+    nodes_attributes = {
+        'node_1': {
+            'shape': np.array([13]),
+            'value': None
+        },
+        'elementwise_node': {
+            'op': None,
+            'kind': 'op',
+            'operation': None
+        },
+        'node_3': {
+            'shape': None
+        }
+    }
+
+    value = np.array([-23.5, -22.5, -2.5, -1.5, -0.5, 0.5, 0.9, 1.5, 2.3, 2.5, 3.5, 22.5, 23.5])
+
+    def test_elementwise_round_even_infer(self):
+        graph = build_graph(self.nodes_attributes,
```

Comment:
Let's move the `build_graph` function to a separate function which is parametrized with a Round `mode`. You will decrease the code size by 2x.

Reply:
Done.